source/hassio/enable_i2c.markdown

```diff
@@ -44,3 +44,20 @@ and make sure the first partition is available.
   present at boot time.
 
 The I2C devices should now be present under /dev.
+
+
+### From Home Assistant Operating System Terminal
+
+At the Home Assistant Operating System Terminal, physical-access, main terminal, 
+  1. login as root
+  2. type `login` to access the shell. 
+  3. type the following to enable i2c. 
+```shell
+mount /dev/sda1 /mnt
+mkdir -p /mnt/CONFIG/modules
+echo -ne i2c-dev>/mnt/CONFIG/modules/rpi-i2c.conf
+echo dtparam=i2c_vc=on >> /mnt/CONFIG/config.txt
+echo dtparam=i2c_arm=on >> /mnt/CONFIG/config.txt
+sync
+reboot
+```
```

Comment:
Indent so it becomes part of the above list.

````suggestion
  ```shell
  mount /dev/sda1 /mnt
  mkdir -p /mnt/CONFIG/modules
  echo -ne i2c-dev>/mnt/CONFIG/modules/rpi-i2c.conf
  echo dtparam=i2c_vc=on >> /mnt/CONFIG/config.txt
  echo dtparam=i2c_arm=on >> /mnt/CONFIG/config.txt
  sync
  reboot
  ```
````